central/pruning/pruning.go

```diff
@@ -144,13 +151,16 @@ func (g *garbageCollectorImpl) pruneBasedOnConfig() {
 	if env.PostgresDatastoreEnabled.BooleanSetting() {
 		postgres.PruneActiveComponents(pruningCtx, globaldb.GetPostgres())
 		postgres.PruneClusterHealthStatuses(pruningCtx, globaldb.GetPostgres())
+
+		g.pruneLogImbues()
 	}
 
 	log.Info("[Pruning] Finished garbage collection cycle")
 }
 
 func (g *garbageCollectorImpl) runGC() {
 	lastClusterPruneTime = time.Now().Add(-24 * time.Hour)
+	lastLogImbuePruneTime = time.Now()
```

Comment:
Look at line 162, I see the purpose of setting the last time back 24 hours is to make sure the prune could happen right at the start up time. It looks like log imbue prune also need it?